src/main/java/seedu/address/model/person/Name.java

```diff
@@ -38,6 +38,21 @@ public static boolean isValidName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    private static String formatName(String name) {
+        String[] words = name.toLowerCase().split("\\s+");
+        StringBuilder formattedWords = new StringBuilder();
+
+        for (String word : words) {
+            if (!word.isEmpty()) {
+                formattedWords.append(Character.toUpperCase(word.charAt(0)))
```

Comment:
I think this block of code need to be abstracted out, it took me quite long to understand what this piece of code is doing

Reply:
Thanks for the suggestion! Ive refactored the code